src/py/flwr/common/differential_privacy_test.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Flower Labs GmbH. All Rights Reserved.
```

Comment:
Since this is a new module, the copyright notices must show the current year:
```suggestion
# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
```